### DISCUSSION TITLE
single exponential sfh

### DISCUSSION THREAD

**Discussion by maciejpkoprowski** [mboquien/cigale#75] [Q&A]
Hi,

I'm just wondering how do you set up SFH history with a single exponential only. 'sfhdelayed' doesn't seem to do the job. Even if I choose f_burst=0, there are still 2 populations produced - stellar.old and stellar.young - the sum of which is then my unattenuated stellar light curve. stellar.young component is not 0, which I assumed it should be given f_burst=0.

Any help would be greatly appreciated,

Mac

**Comment by mboquien** [comment]
Hello. The young/old indices are to separate the spectra compute the differential reddening. There is no incidence on the number of populations.

**Comment by maciejpkoprowski** [comment]
So, the SFH has only main stellar population when f_burst=0? There is no young burst? The SFH has only 1 exp component?

**Comment by mboquien** [comment]
For `sfhdelayed`, it would be a delayed component (SFR∝exp(-t/τ)). If you want a pure exponential, the `sfh2exp` module with the `f_burst` parameter set to 0 would be more appropriate.

**Comment by maciejpkoprowski** [comment]
Isn't sfh2exp with f_burst=0 same as sfhdelayed with f_burst=0, given age_main and tau_main are the same in both cases?

**Comment by mboquien** [comment]
No, one is an exponential SFH, the other one a delayed SFH.
